packages/tenants/README.md

```diff
@@ -6,7 +6,7 @@
     height="250px"
   />
 </p>
-<h1 align="center"><b>Credo - Tenant Module</b></h1>
+<h1 align="center"><b>Credo Tenant Module</b></h1>
```

Comment:
Is it 'Tenant' or 'Tenants'?

Reply:
good question :)

Reply:
probably tenants

Reply:
I like those auto-suggestions. It remembers me the auto-passes in football 😄